src/utils/webrtc/models/CallParticipantModel.js

```diff
@@ -121,8 +122,16 @@ CallParticipantModel.prototype = {
 		this._trigger('change:' + key, [value])
 	},
 
+	// Compare Object and Proxy(Object), whether they refer to the same object
+	_isSameProxy(obj1, obj2) {
+		if (typeof obj1 !== 'object' || obj1 === null || typeof obj2 !== 'object' || obj2 === null) {
+			return obj1 === obj2
+		}
+		return Reflect.getPrototypeOf(obj1) === Reflect.getPrototypeOf(obj2)
+	},
```

Comment:
Vue does not create a proxy twice for the same object...
Also, Vue has `toRaw` function to get the original object from Proxy.

Doesn't `return toRaw(obj1) === toRaw(obj2)` work here?

Reply:
Oh, I missed that we store Peer in reactive data. Probably we should store it as a non-reactive, raw value using `toRaw` to not have proxy created for Peer at all.